
<div class="mainWrapper">
  <div class="col-left">
    
  </div>
  <div class="col-right">

  <img src="/group_logo.png" alt="group_logo" style="max-width: 50%">
  
  </div>
</div> 


# Research

*My research interests cover various topics of microbial genomics from development of new methodology and their application for medical purposes to fundamental questions of genome organization and evolution. Current projects I am working on are:*

## Detecting the impact of large-scale genomic features on bacterial phenotypes

<p style="margin-top: 15px;">
<div class="col-left">
  <img src="/Research/RearrangementsWolbachia.png" alt="Research picture" style="max-width: 90%">
</div>
</p>

**The main goal of the project is to connect the structural variants with phenotypic changes and to understand the underlying molecular mechanisms connecting them.** There are two main processes that generate the variability that can be leveraged to explore the genotype to phenotype connection: parallel adaptation to new environments (e.g. new ecological niche or host organism) and small colony variation (sustaining variability of phenotypes in pathogenic populations to avoid the host organism’s immune system response).

*The project is supported by the [FWF grant #ESP 253-B](https://pf.fwf.ac.at/en/research-in-practice/project-finder/58815)*

## Role of gene paralogs in bacterial chromosome maintainance

<p style="margin-top: 15px;">
<div class="col-left">
  <img src="/Research/repeats.png" alt="Research picture" style="max-width: 90%">
</div>
</p>

**The project aims to investigate the patterns on genomic repeats across the circular bacterial chromosome to reveal interplay between chromosome topology and gene paralogization.**
Copy number variation is the important genomic trait associated with bacterial phenotype. In particular, number of rRNA gene operons is species-specific and supposed to be associated with ecological niches. On the other hand, such genomic repeats provide substrates for intra-genomic recombination leading to genome rearrangements. We assume that recombination events as well as composition of genomic repeats are shaped by selection forces balancing profit and damages on different levels of chromosome organization.

*The project is supported by the [FWF grant #ESP 253-B](https://pf.fwf.ac.at/en/research-in-practice/project-finder/58815)*

## Phase variation of susC genes in Bacteroides
<sup>*</sup>A collaboration with Prof. David Berry, CMESS, UniWien, Austria

<p style="margin-top: 15px;">
<div class="col-left">
  <img src="/Research/bacteroides.png" alt="Research picture" style="max-width: 90%">
</div>
</p>

**The project aims to investigate the molecular mechanism of phase variation producing shuffling of susC genes in Bacteroides**
Bacteroides are a part of the human gut microbiota with an extensive capacity for glycan harvest. These bacteria express the starch utilization system (Sus) in the outer membrane, which binds, degrades, and imports starch into the cell. Bacteroides can alter the expression of SusC/SusD family outer membrane proteins and capsular polysaccharides through reversible DNA inversions, putatively driven by site-specific recombinases. The main goal of the project is to elucidate the mechanism of phase variation of susC proteins and shed light on its impact on the ability to degrade different glycans in host diets.

## Role of non-canonical start codons in bacteria
<sup>*</sup>A collaboration with Prof. Calin Guet, IST Austria

<p style="margin-top: 15px;">
<div class="col-left">
  <img src="/Research/startcodons.png" alt="Research picture" style="max-width: 90%">
</div></p>

**The project is aimed to reconstruct the evolution of start codons in bacterial genes** While _ATG_ starts are most common among bacteria genes, _TTG_ and _GTG_ codons are also present in genomes. For example, an analysis of 620 bacterial genomes revealed that ~80% of annotated genes initiate at AUG codons, ~12% at GUG and ~8% at UUG, with variable incidences of AUU and AUC across species [citation](https://www.ncbi.nlm.nih.gov/pmc/articles/PMC7256928/). In our research we reconstruct the evolution of genes with weak start codons and analyse the functions of these genes. 

## Machine learning and phylogenetic analysis for prediction of antibiotic resistance
<sup>*</sup>A collaboration with Prof. Olga Kalinina, HIPS Germany

<p style="margin-top: 15px;">
<div class="col-left">
  <img src="/Research/ML.png" alt="Research picture" style="max-width: 90%">
</div></p>

**The project is aimed to develop ML models for discovery of antibiotics resistance markers.** We present ML models for the discovery of antibiotic resistance markers. The models are trained using whole-genome sequences with accompanying resistance screens, and the resistance markers are extracted with feature importance analysis. We emphasize the importance of accounting for population structure within a bacterial species by introducing PRPS, phylogeny-related parallelism score. We show that ML models that employ PRPS-aware features demonstrate superior performance, as well as discover more biologically meaningful markers.

[paper](https://doi.org/10.1186/s12866-023-03147-7)

## Fitness effects of short random peptides
<sup>*</sup>A collaboration with Prof. Dan I. Andersson, Uppsala University and Dr. Roderich Roemhild, IST Austria

<p style="margin-top: 15px;">
<div class="col-left">
  <img src="/Research/Biological replicate II trends.png" alt="Research picture" style="max-width: 90%">
</div></p>

**The project is aimed to estimate the distribution of fitness effects of random non-coding DNA in microbial cells.** It is generally assumed that new genes arise through duplication and/or recombination of existing genes. Previous experimental work confirmed that new functional genes could arise out of random non-coding DNA. In our research we estimate the fitness effects of de-novo transcribed random DNA in *E. coli*.

## Development of bioinformatic tools for detection and analysis of large-scale genomic variants in bacterial genomes ###

<sup>*</sup>A collaboration with Dr. Nikita Alexeev

<p style="margin-top: 15px;">
<div class="col-left">
  <img src="/Research/pipeline-1-1.png" alt="Research picture" style="max-width: 90%">
</div>
</p>

**The project aims to develop bioinformatic software to detect large-sclae genomic variants in bacterial genomes and linked them with bacterial phenotypes.**
We have developed a strategy for extract information form genomic data to detect parallel
rearrangements in bacterial populations. The approach will be used for the study of rapid emergence of new bacterial phenotypes, understanding the molecular basis of antibiotic resistance mechanisms and formation of small colony variants, and the study of the selective forces in genomic evolution underlying complex phenotypes. The application of this approach and the concomitant understanding of connections between detected genome rearrangements
and medically-relevant phenotypes may contribute to the efficient development of drugs and vaccines.

[The PaReBrick toolkit](https://github.com/ctlab/parallel-rearrangements) [Paper](https://academic.oup.com/bioinformatics/advance-article/doi/10.1093/bioinformatics/btab691/6380551)


## Origin and evolution of the multi-chromosome bacterial genomes

<sup>*</sup>A collaboration with Prof. Mikhail Gelfand

<p style="margin-top: 15px;">
<div class="col-left">
  <img src="/Research/burkholderia.png" alt="Research picture" style="max-width: 90%">
</div>

**The project aims to understand organization and evolutionary benefits of bacterial genomes with secondary replicons.**
Most bacterial genomes have a single chromosome that may be supplemented by a few smaller,
dispensable plasmids. But approximately 10% of the bacteria with completely sequenced genome,
mostly pathogens and plant symbionts, have essential megaplasmids and/or chromids.  However, the advantages of multi­chromosomal genome organiza­tion remain unclear.
</p>

*The project was supported by the Marie Skłodowska­Curie Grant No. 754411.*

[Conference paper](http://mccmb.belozersky.msu.ru/2021/thesis/abstracts/93_MCCMB_2021.pdf), [Preprint](https://www.biorxiv.org/content/10.1101/2023.04.09.536151v2)


## Evolution of virulence factors in human-host and non-human-host invasive *Escherichia* 
<p style="margin-top: 15px;">
<div class="col-left">
  <img src="/Research/Shigella_animals.jpg" alt="Research picture" style="max-width: 90%">
</div></p>

**The project aims to describe composition and evolution of *ipaH* genes, effectors of Type 3 secretion system, in pathogens of different hosts.**
These genes are key factors of *Shigella* invasion that which are used for disease genotyping. Until recently, *Shigella* were thought to be primate-restricted pathogens. However, recent genomic studies confirmed *ipaH* genes in genome of *Escherichia marmotae*, a potential marmot pathogen, and of an *E. coli* extracted from fecal samples of bovine calves, suggesting that non-human hosts may also be infected by these potentially pathogenic to humans strains. We employ a computational approach to predict whether different *Escherichia* may also be an infectious agent of non-human hosts, which, therefore, may serve as a reservoir of human pathogens and virulence genes.

[Paper]((https://doi.org/10.1038/s41598-022-10827-3))

## Сooperation partners:
2023-present **Prof. David Berry**, University of Vienna, Vienna, **Austria**. Project: Identification of shafflons in gut microbiomes.
2022-present **Prof. Fyodor Kondrashov**, OIST, Okinawa, **Japan**. Project: Selection forces in multipartite bacterial genomes.
2022-present **Prof. Calin Guet** IST Austria, Klosterneuburg, **Austria**. Project: Conservation of non-canonical start codons in mar operon genes.
2022-present **Prof. Olga Kalinina**, Helmholtz Institute for Pharmazeutical Research Saarland (HIPS), **Germany**. Project: Machine learning and phylogenetic analysis improves predicting antibiotic resistance in M. tuberculosis
2021-present **Prof. Dan I. Andersson**, Uppsala University, **Sweden**. Project: Fitness effects of short random peptides.
2021-present **Prof. Mikhail Gelfand**, IITP RAS, **Russia**. Project: Evolution of multi-partite genomes.
2020-2022 **Dr. Nikita Alexeev**, ITMO University, Saint Petersburg, **Russia**. Project: Development of the bioinformatic toolkit for whole-genome analysis.
2020-2022 **Prof. Christoph Gasche**, Medical University of Vienna, Vienna, **Austria**. Project: Genetic factors under biofilms formation in pathogenic E.coli.
2016-2018 **Prof. Marc Robinson-Rechavi**, Evolutionary Bioinformatics Lab, Department of Ecology and Evolution, Université de Lausanne, Lausanne, **Switzerland**. Project: Positive selection and horizontal gene transfer in prokaryotes.
2010-2014 **Prof. Pavel Pevzner**, University of California at San Diego, California, **USA**. Project: Application of the MGRA (Multiple Genome Rearrangements and Ancestors) algorithm to microbial data.
