---
title: "Research"
---
*My research interests cover various topics of microbial genomics from development of new methodology and their application for medical purposes to fundamental questions of genome organization and evolution. Current projects I am working on are:*

## Evolution of start codons

<p style="margin-top: 15px;">
<div class="col-left">
  <img src="/Ecoli starts per genome.pdf" alt="Research picture" style="max-width: 90%">
</div></p>

**The project is aimed to reconstruct the evolution of start codons in bacterial genes** While _ATG_ starts are most common among bacteria genes, _TTG_ and _GTG_ codons are also present in genomes. For example, an analysis of 620 bacterial genomes revealed that ~80% of annotated genes initiate at AUG codons, ~12% at GUG and ~8% at UUG, with variable incidences of AUU and AUC across species [citation](https://www.ncbi.nlm.nih.gov/pmc/articles/PMC7256928/). In our research we reconstruct the evolution of genes with weak start codons and analyse the functions of these genes. 

## Fitness effects of short random peptides
<sup>*</sup>A collaboration with Dr. Roderich Кromhild, IST Austria

<p style="margin-top: 15px;">
<div class="col-left">
  <img src="/Biological replicate II trends.png" alt="Research picture" style="max-width: 90%">
</div></p>

**The project is aimed to estimate the distribution of fitness effects of random non-coding DNA in microbial cells.** It is generally assumed that new genes arise through duplication and/or recombination of existing genes. Previous experimental work confirmed that new functional genes could arise out of random non-coding DNA. In our research we estimate the fitness effects of de-novo transcribed random DNA in *E. coli*.

## Development of bioinformatic tools for detection and analysis of large-scale genomic variants in bacterial genomes ###

<sup>*</sup>A collaboration with Dr. Nikita Alexeev

<p style="margin-top: 15px;">
<div class="col-left">
  <img src="/pipeline-1-1.png" alt="Research picture" style="max-width: 90%">
</div>

**The project aims to develop bioinformatic software to detect large-sclae genomic variants in bacterial genomes and linked them with bacterial phenotypes.**
We have developed a strategy for extract information form genomic data to detect parallel
rearrangements in bacterial populations. The approach will be used for the study of rapid emergence of new bacterial phenotypes, understanding the molecular basis of antibiotic resistance mechanisms and formation of small colony variants, and the study of the selective forces in genomic evolution underlying complex phenotypes. The application of this approach and the concomitant understanding of connections between detected genome rearrangements
and medically-relevant phenotypes may contribute to the efficient development of drugs and vaccines.

</p>

[The PaReBrick toolkit](https://github.com/ctlab/parallel-rearrangements) [Paper](https://academic.oup.com/bioinformatics/advance-article/doi/10.1093/bioinformatics/btab691/6380551)


## Origin and evolution of the multi-chromosome bacterial genomes

<sup>*</sup>A collaboration with Prof. Mikhail Gelfand and Prof. Fyodor Kondrashov

<p style="margin-top: 15px;">
<div class="col-left">
  <img src="/burkholderia.png" alt="Research picture" style="max-width: 90%">
</div>

**The project aims to understand organization and evolutionary benefits of bacterial genomes with secondary replicons.**
Most bacterial genomes have a single chromosome that may be supplemented by a few smaller,
dispensable plasmids. But approximately 10% of the bacteria with completely sequenced genome,
mostly pathogens and plant symbionts, have essential megaplasmids and/or chromids.  However, the advantages of multi­chromosomal genome organiza­tion remain unclear.
</p>

*The project is supported by the European Union’s Horizon 2020 Research and Innovation Programme
under the Marie Skłodowska­Curie Grant Agreement No. 754411.*

[Conference paper](http://mccmb.belozersky.msu.ru/2021/thesis/abstracts/93_MCCMB_2021.pdf)

## Role of gene paralogs in bacterial chromosome maintainance

<p style="margin-top: 15px;">
<div class="col-left">
  <img src="/repeats.png" alt="Research picture" style="max-width: 90%">
</div>

**The project aims to investigate the patterns on genomic repeats across the circular bacterial chromosome to reveal interplay between chromosome topology and gene paralogization.**
Copy number variation is the important genomic trait associated with bacterial phenotype. In particular, number of rRNA gene operons is species-specific and supposed to be associated with ecological niches. On the other hand, such genomic repeats provide substrates for intra-genomic recombination leading to genome rearrangements. We assume that recombination events as well as composition of genomic repeats are shaped by selection forces balancing profit and damages on different levels of chromosome organization.

</p>

## Evolution of virulence factors in human-host and non-human-host invasive *Escherichia* 
<p style="margin-top: 15px;">
<div class="col-left">
  <img src="/Shigella_animals.jpg" alt="Research picture" style="max-width: 90%">
</div></p>

**The project aims to describe composition and evolution of *ipaH* genes, effectors of Type 3 secretion system, in pathogens of different hosts.**
These genes are key factors of *Shigella* invasion that which are used for disease genotyping. Until recently, *Shigella* were thought to be primate-restricted pathogens. However, recent genomic studies confirmed *ipaH* genes in genome of *Escherichia marmotae*, a potential marmot pathogen, and of an *E. coli* extracted from fecal samples of bovine calves, suggesting that non-human hosts may also be infected by these potentially pathogenic to humans strains. We employ a computational approach to predict whether different *Escherichia* may also be an infectious agent of non-human hosts, which, therefore, may serve as a reservoir of human pathogens and virulence genes.

[Paper]((https://doi.org/10.1038/s41598-022-10827-3))

## Other collaborative projects:
- ***(2022-present)*** **Evolution of _maRAB_ operon in _gammaproteobacteria_** with Dr. Kirti Jain, IST Austria.
- ***(2020-2022)*** **Genetic factors under biofilms formation in pathogenic _E.coli_**
with Prof. Christoph Gasche, Medical University of Vienna, Austria and Dr. Maria Tutukina, Institute of Cell Biophysics RAS, Russia
- ***(2016-2018)*** **Positive selection and horizontal gene transfer in prokaryotes** with Prof. Marc Robinson-Rechavi, Université de Lausanne, Lausanne, Switzerland, supported by the Russian Foundation of Basic Research and Swiss National Science Foundation, joint grant
